recover_affected_workloads.md

```diff
@@ -0,0 +1,57 @@
+# Recover Affected Workloads
+
+In kubernetes world there is no concept of rescheduling. When pods are bounded to a
```

Comment:
I don't think it's true. the scheduler can move pods from node to node

Reply:
can you put the link with the doc here?

Reply:
I don't follow. which doc?